shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeIngressService.java

```diff
@@ -1,9 +1,14 @@
 package com.redhat.service.bridge.shard.operator;
 
 import com.redhat.service.bridge.infra.models.dto.BridgeDTO;
+import com.redhat.service.bridge.shard.operator.resources.BridgeIngress;
+
+import io.fabric8.kubernetes.api.model.apps.Deployment;
 
 public interface BridgeIngressService {
     void createBridgeIngress(BridgeDTO bridgeDTO);
 
     void deleteBridgeIngress(BridgeDTO bridgeDTO);
```

Comment:
nit; Maybe we can consider changing this signature in the future since it's a `BridgeIngressService` and receive a `BridgeIngress` reference instead.